.github/workflows/ci.yml

```diff
@@ -204,7 +204,14 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --all-features --all-targets -- -D warnings
 
-  fmt:
+      # This check makes sure the crate dependency check is accurate
+      - name: Check Cargo.lock is up to date
+        uses: actions-rs/cargo@v1.0.3
+        with:
+          command: check
+          args: --locked --all-features --all-targets
```

Comment:
Should this step go in the `Clippy` job?

Reply:
We're having trouble with long CI run times. This is the most efficient place to put the lockfile check, because it re-uses the crates that were downloaded by clippy. So it completes really quickly.

Reply:
I opened #2995 to review jobs and steps once CI is faster.

Reply:
See my answer to the same question here https://github.com/ZcashFoundation/zebra/pull/2986#discussion_r740497396